docs/CLI_Reference/index.md

```diff
@@ -31,13 +31,13 @@ Use "oras [command] --help" for more information about a command.
 
 ## See also
 
-* [oras attach](0_oras_attach.md)	 - Attach files to an existing artifact
+* [oras attach](oras_attach.md)	 - Attach files to an existing artifact
 * oras blob - Blob operations
-    * [oras blob delete](1_oras_blob_delete.md)	 - Delete a blob from a remote registry
-    * [oras blob fetch](2_oras_blob_fetch.md)	 - Fetch a blob from a remote registry
-    * [oras blob push](3_oras_blob_push.md)	 - Push a blob to a remote registry
-    * [oras copy](4_oras_copy.md)	 - Copy artifacts from one target to another
-* [oras discover](5_oras_discover.md)	 - Discover referrers of a manifest in the remote registry
+    * [oras blob delete](oras_blob_delete.md)	 - Delete a blob from a remote registry
+    * [oras blob fetch](oras_blob_fetch.md)	 - Fetch a blob from a remote registry
+    * [oras blob push](oras_blob_push.md)	 - Push a blob to a remote registry
```

Comment:
I found those descriptions are not accurate, e.g. `blob fetch` and `blob push` can be used to operate OCI layouts. Will create a PR to update the doc in ORAS CLI, for this PR I am okay to merge it this way.